src/oc_bifrost/apps/bifrost/src/bifrost_db.erl

```diff
@@ -88,6 +90,18 @@ exists(Type, AuthId) ->
     {ok, Answer} = select(StatementName, [AuthId], first_as_scalar, [exists]),
     Answer.
 
+get_id_query(actor)     -> get_actor_id;
+get_id_query(container) -> get_container_id;
+get_id_query(group)     -> get_group_id;
+get_id_query(object)    -> get_object_id.
+
+-spec get_id(auth_type(), auth_id()) -> not_found | boolean().
```

Comment:
This type is definitely weird.  get_id_query returns a bigserial, not an auth_id() (which is a string).